src/components/picker/picker.tsx

```diff
@@ -349,17 +345,20 @@ export class Picker {
         return this.items.length > ITEM_LIMIT_NO_SCROLL;
     }
 
-    private renderPortal(content = null, styling = {}) {
+    private renderPortal(content = null) {
         return (
             <limel-portal
                 visible={!!content}
                 containerId={this.portalId}
-                containerStyle={styling}
                 inheritParentWidth={true}
             >
                 <limel-menu-surface
                     open={!!content}
-                    style={{ '--menu-surface-width': '100%' }}
+                    style={{
+                        '--menu-surface-width': '100%',
+                        'max-height': 'inherit',
+                        display: 'flex',
```

Comment:
I was just about to comment that these should be set in the scss file, but that doesn't work because of the portal I guess?

Reply:
I tried to fix this with Kia but we didn't manage to get it to work. It's because it is set to the container so it is not in the right place in the HTML to be in the Portal's scss file. But I guess we could set it in the portal.tsx file 🤔

Reply:
No, don't worry about it. It's much better that it's set here than in the portal.